package.json

```diff
@@ -31,8 +31,8 @@
     "browserDependencies": "all"
   },
   "dependencies": {
-    "chart.js": "2.7.3",
-    "chartjs-plugin-streaming": "1.7.1"
+    "chart.js": "2.8.0",
```

Comment:
Are you thinking it is best to pin versions for these modules?  Since there are so many moving parts to chart.js (including css) it does seem like we might want to test any updates to ensure there is no breakage.

Reply:
only issue is zoom and pan did not check if it updated or works with latest update. subscribed to releases for both chart.js and streaming plugin.

Reply:
ps this has not been tested in Safari and firefox so will do tonight.

Reply:
also fresh clone needs to be tested too cache is still there. last time that was the issue.